controllers/users.js

```diff
@@ -0,0 +1,114 @@
+const { User } = require('../models')
+const bcrypt = require('bcryptjs')
+const jwt = require('jsonwebtoken')
+require('dotenv').config()
+
+class Controller {
+    static showAll(req,res){
+        User.findAll({order:[["id","ASC"]]})
+        .then((users)=>{
+            res.json(users)
+        })
+        .catch((err)=>{
+            res.status(500).json(err)
+        })
+    }
+    static showOne(req,res){
+        let id = req.params.id
+        User.findByPk(id)
+        .then((user)=>{
+            res.json(user)
+        })
+        .catch((err)=>{
+            res.status(500).json(err)
+        })
+    }
+    static add(req,res){
+        let username = req.body.username
+        let password = bcrypt.hashSync(req.body.password, 8)
+        let role = req.body.role
+        User.create({username:username,password:password,role:role})
```

Comment:
Indentasinya harus dirapihkan

```suggestion
        User.create({username: username, password: password, role: role})
```

Jangan pelit pelit spasi